### PR TITLE
docs: Create CONTRIBUTING

### PR DESCRIPTION
**Pull request by loic5**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+If you have any questions, create an [issue](https://github.com/TypedProject/tsed/issues) (protip: do a quick search first to see if someone else didn't ask the same question before!). You can also reach us at https://gitter.im/Tsed-io/community.
+
+## How to work on Ts.ED
+### Setup
+
+Clone your fork of the repository
+
+```bash
+$ git clone https://github.com/YOUR_USERNAME/tsed.git
```
Comment (Romakita) [inline CONTRIBUTING.md:50]: git clone https://github.com/YOUR_USERNAME/tsed.git 

A remplacer par:
git clone https://github.com/YOUR_USERNAME/tsed-cli.git

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+```
+
+Install npm dependencies with yarn (not with NPM!):
+```bash
+yarn
+```
+> After installing dependencies, yarn/npm run the `postinstall` hook and mounted all packages with `npm link` (e.g. `yarn run repo:bootstrap`).
```
Comment (Romakita) [inline CONTRIBUTING.md:57]: Tu peux supprimer cette ligne. :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+```bash
+npm install -g gflow
+```
+
+### Start a feature branch
+
+```bash
+git fetch
+git branch --no-track -b feat-branch-name origin/production  # !IMPORTANT
```
Comment (Romakita) [inline CONTRIBUTING.md:89]: production => master

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+yarn
+
+## OR
+gflow new feat name_of_feat
+```
+
+### Commit & Push a feature
+
+This command rebases your branch feature from the production branch, runs the test, and pushes your branch.
```
Comment (Romakita) [inline CONTRIBUTING.md:98]: production => master

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+```bash
+git commit -m "feat(domain): Your message"
+```
+
+Then:
+```bash
+npm run test
+git fetch
+git rebase origin/production
```
Comment (Romakita) [inline CONTRIBUTING.md:108]: production => master

**Review by Romakita — CHANGES_REQUESTED** [review]

**Comment by Romakita** [comment]
Thanks Loïc :)
There are small requested change. Ask me tomorrow to know how you can apply change on the current PR :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (loic5) [reply CONTRIBUTING.md:50]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (loic5) [reply CONTRIBUTING.md:57]: done

File: CONTRIBUTING.md (hunk 3 above)
Comment (loic5) [reply CONTRIBUTING.md:89]: done

File: CONTRIBUTING.md (hunk 4 above)
Comment (loic5) [reply CONTRIBUTING.md:98]: done

File: CONTRIBUTING.md (hunk 5 above)
Comment (loic5) [reply CONTRIBUTING.md:108]: done

**Review by Romakita — APPROVED** [review]

**Pull request merged**